src/main/java/com/exactpro/th2/FixHandler.java

```diff
@@ -180,9 +171,20 @@ public FixHandler(IHandlerContext context) {
                 scheduleTime = now.plusDays(1).with(resetTime);
             }
             long time = scheduleTime.atZone(ZoneId.systemDefault()).toInstant().toEpochMilli() - System.currentTimeMillis();
-            executorService.schedule(this::reset, time, TimeUnit.MILLISECONDS);
-        } else {
-            resetTime = LocalTime.now();
+            executorService.scheduleAtFixedRate(() -> reset(settings.getSessionEndTime() == null), time, 24, TimeUnit.HOURS);
```

Comment:
> settings.getSessionEndTime() == null

I think we should make `sessionEndTime` mandatory if `sessionStartTime` is present

> reset(settings.getSessionEndTime() == null)

We shouldn't not only reset seqnums at start time but also send login

> time, 24, TimeUnit.HOURS

`time` is in milliseconds